geomeppy/view_geometry.py

```diff
@@ -79,6 +79,27 @@ def view_polygons(polygons):
 
     plt.show()
 
+	
+def _get_geometry_rules(idf):
+    """Get the geometry rules from the IDF
+    """
+    rule_types = ["GLOBALGEOMETRYRULES"]
+    rule = []
```

Comment:
I prefer list comprehensions in this kind of case.

`rule = [idf.idfobjects[rule_type] for rule_type in rule_types]`

Reply:
@jamiebull1 if I use a list comprehension, items in the list get converted to `'Idf_MSequence` objects instead of a `EpBunch`. Do you know why? `_get_geometry_rules(idf)[0].Coordinate_System.lower()` results in an error.

I don't understand why the object type changes whether we use `.extend` or a list comprehension.

Reply:
Not sure. Possibly something `eppy`-related. I'll try and look into it if I get a chance.

Reply:
I asked [here](https://github.com/santoshphilip/eppy/issues/234), I understand now, `EpBunch` objects are index dependent, `Idf_MSequence` aren't. `rule = [idf.idfobjects[rule_type][0] for rule_type in rule_types` produces a `EpBunch`. I'll edit the code.